packages/client-core/src/admin/DefaultAdminRoutes.tsx

```diff
@@ -150,5 +153,12 @@ export const DefaultAdminRoutes: Record<string, AdminRouteStateType> = {
     component: Channels,
     access: false,
     icon: <HiOutlineMegaphone />
+  },
+  crashReport: {
+    name: 'user:dashboard.crashReport',
+    scope: 'crashReport',
```

Comment:
We dont have `'crashReport'` in our scope types. You can use `server` here

Reply:
done b16316005f9954108a279bc4781f33bd5357a99b